### ISSUE TITLE
Gradients in IE

### ISSUE THREAD

**Issue by athomann** [LeaVerou/prefixfree#80]
I noticed this gradient, in IE, doesn't show the correct degree angle unless I use a prefix.

background: linear-gradient(180deg,  #222323 0%,#1c1c1e 80%, #fff 82%, #090909 75%,#000000 100%);
background: -ms-linear-gradient(180deg,  #222323 0%,#1c1c1e 80%, #fff 82%, #090909 75%,#000000 100%);

**Comment by LeaVerou** [comment]
Yup, it's because IE unprefixed gradients, and the current spec uses different angles than it used to, so they had to follow that one.
I will have to think how to address this problem in the best way for -prefix-free users. Expect an update at some point.

**Comment by mani313** [comment]
Yes It sucks! in IE for Gradient....i tried it
background: linear-gradient(#444,#222); background: -ms-linear-gradient(#444,#222); 

it doesn't work on IE9 is the code have to change?

**Comment by LeaVerou** [comment]
IE9 doesn't support CSS gradients. I was talking about IE10.
